x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -34,11 +36,14 @@ const TestProvidersComponent: React.FC<Props> = ({
   children,
   appId = 'testAppId',
   userCanCrud = true,
+  rootBreadcrumbs,
```

Comment:
I think it would be nice to have a default here also. Something like: `rootBreadcrumbs = [{ text: 'Test' }]`

Reply:
This prop is actually optional, if not passed the breadcrumbs can be managed from outside Cases. I wanted to test that if `rootBreadcrumbs` is undefined the breadcrumbs are not controlled by us.
Do you think it is better to make it required and always manage the breadcrumbs?

Reply:
I think that if the majority of the tests need `rootBreadcrumbs` then it is nice to have a default to avoid having to pass it for every test. You can put `rootBreadcrumbs={undefined}` on the test you want to check that behavior.

>Do you think it is better to make it required and always manage the breadcrumbs?

I am not sure. I think is best to have total control by making it required and leave any ambiguity (and potential bugs) to the consumers of our API